server/lib/template/helpers/finance.js

```diff
@@ -30,6 +30,26 @@ function currency(value = 0, currencyId) {
   return new Handlebars.SafeString(accountingjs.formatMoney(value, fmt));
 }
 
+/**
+ * @function currencyWithPosition
```

Comment:
I wonder if a different name might be more descriptive:  currencyWithPrecision, currencyPrecision, ...?